service/matching/backlog_manager_test.go

```diff
@@ -42,6 +41,8 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+/* TODO: I can't figure out what these tests are testing
```

Comment:
If you don't see a use for these either, I can just delete them in this PR

Reply:
👍 to delete